crates/aide/src/axum/mod.rs

```diff
@@ -192,6 +192,9 @@ use indexmap::IndexMap;
 use tower_layer::Layer;
 use tower_service::Service;
 
+#[cfg(feature = "axum-extra")]
+use axum_extra::routing::RouterExt as AxumExtraRouterExt;
```

Comment:
Since this extension trait is never named, doesn't need to have a local alias:

```suggestion
use axum_extra::routing::RouterExt as _;
```

Reply:
Nice to learn about this technique